lib/routes/emails.js

```diff
@@ -491,12 +494,15 @@ module.exports = (log, db, mailer, config, customs, push) => {
         const sessionToken = request.auth.credentials
         const uid = sessionToken.uid
 
-        if (! features.isSecondaryEmailEnabled(sessionToken.email)) {
-          return reply(error.featureNotEnabled())
-        }
+        return db.account(uid)
+          .then((account) => {
 
-        db.accountEmails(uid)
```

Comment:
@philbooth Added some more commentary on how this [works](https://github.com/mozilla/fxa-auth-server/blob/dfbbb0fdd216ed803a6580299492b54fed6da81d/lib/routes/account.js#L475), does it kind of clear it up?